packages/client/README.md

```diff
@@ -20,27 +38,25 @@ There are three ways to use the SDK:
 
 ### Schema-generated Client
 
-To use the schema-generated client, you need to run the code generator utility that comes with [our CLI](../../cli/README.md).
+To use the schema-generated client, you need to run the code generator utility that comes with [our CLI](https://docs.xata.io/cli/getting-started).
 
 To run it (and assuming you have configured the project with `xata init`):
 
 ```bash
 xata codegen
 ```
 
-In a TypeScript file start using the generated code:
+In a TypeScript file, start using the generated code like this:
 
 ```ts
-import { XataClient } from './xata';
+import { XataClient } from './xata'; // or wherever you chose to generate the client
 
-const xata = new XataClient({
-  fetch: fetchImplementation // Required if your runtime doesn't provide a global `fetch` function.
-});
+const xata = new XataClient();
 ```
 
-The import above will differ if you chose to genreate the code in a different location.
+The import above will differ if you chose to generate the code in a different location.
 
-The `fetch` paramter is required only if your runtime doesn't provide a global `fetch` function. There's also a `databaseURL` argument that by default will contain a URL pointing to your database (e.g. `https://myworkspace-123abc.xata.sh/db/databasename`), it can be specified in the constructor to overwrite that value if for whatever reason you need to connect to a different workspace or database.
+The constructor `XataClient` accepts an object with configuration options, like the `fetch` parameter, which is required only if your runtime doesn't provide a global `fetch` function. There's also a `databaseURL` argument that by default will contain a URL pointing to your database (e.g. `https://myworkspace-123abc.xata.sh/db/databasename`). It can be specified in the constructor to overwrite that value if for whatever reason you need to connect to a different workspace or database.
```

Comment:
```suggestion
The `XataClient` constructor accepts an object with configuration options, like the `fetch` parameter, which is required only if your runtime doesn't provide a global `fetch` function. There's also a `databaseURL` argument that by default will contain a URL pointing to your database (e.g. `https://myworkspace-123abc.xata.sh/db/databasename`). It can be specified in the constructor to overwrite that value if for whatever reason you need to connect to a different workspace or database.
```